company.el

```diff
@@ -64,6 +64,7 @@
 (require 'cl-lib)
 (require 'newcomment)
 (require 'pcase)
+(require 'find-func)
```

Comment:
Why `find-func`?

Should it be `subr-x` instead? Which your addition of the use of `if-let*` seems to require.

This will also require a dependency bump to at least 25.1. There is still no `if-let*` in `subr-x` in 25.1, but IIUC `if-let` (no asterisk) behaves identically.

Reply:
It is for `find-library-name`.

I will change if-let* to if-let

Reply:
> It is for find-library-name

Ah, ok.

You can do the same thing a bit faster without it. See: https://github.com/company-mode/company-mode/blob/master/company-tests.el#L24-L25